packages/vuetify/test/unit/components/VTimePicker/VTimePickerTitle.spec.js

```diff
@@ -2,126 +2,147 @@ import VTimePickerTitle from '@/components/VTimePicker/VTimePickerTitle'
 import { test } from '@/test'
 
 test('VTimePickerTitle.js', ({ mount }) => {
-  it('should render component in 24hr', () => {
-    const wrapper = mount(VTimePickerTitle, {
-      propsData: {
-        hour: 14,
-        minute: 13,
-        period: 'pm',
-        ampm: false
-      }
+  [true, false].forEach(useSecondsValue => {
```

Comment:
I was tested that option useSeconds not affect the rest of the functionality and everything works regardless of the value useSeconds. You offer all to test with useSeconds = false and separately to test only where this parameter is affected?

Reply:
I understand the reason, however if we want to use this approach we should test all combinations of props, because every prop may have impact on something
Maybe just leave it for now (or change to `describe.each` as Kael suggested), if it will become a problem we'll handle it later

Reply:
I've never worked with unit tests before. TypeError: describe.each is not a function